index.html

```
@@ -65,6 +66,37 @@ <h2 class="movie-section-title"></h2>
         <p>&copy; 우아한테크코스 All Rights Reserved.</p>
       </footer>
     </div>
+    <div class="modal-background" id="modalBackground">
+      <div class="modal">
+        <button class="close-modal" id="closeModal">
+          <img src="./images/modal_button_close.png" alt="" aria-hidden="true" />
+        </button>
+        <div class="modal-container">
+          <div class="modal-image">
+            <img id="modal-poster" src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)